content/version/2/0/code_of_conduct.fr.md

```diff
@@ -0,0 +1,132 @@
+# Charte Code de Conduite Contributeurs
+
+## Notre engagement
+
+En tant que membres, contributeurs et dirigeants, nous nous 
+engageons à rendre la participation dans notre communauté 
+une expérience sans harcelement pour tous, quel que soit l'âge, 
+la physionomie, la présence d'un handicap visible ou invisible, l'ethnie, 
+les caractéristiques sexuelles, l'identité et l'expression de genre, 
+le niveau d'expérience, l'éducation, le statut socio-économique, 
+la nationalité, l'apparence personnelle, la race, la religion, 
+ou l'identité et l'orientation sexuelle.
+
+Nous nous engageons à agir et interagir de manière à contribuer à une communauté 
+ouverte, accueillante, diverse, inclusive et saine.
+
+## Nos critères
+
+Exemples de comportements qui contribuent à créer un environnement positif :
+
+* Faire preuve d'empathie et de bienvaillance envers les autres membres
+* Le respect des différents points de vue, opinions et expériences vécues
+* Donner et accepter poliment les critiques constructives
+* Assumer ses responsabilités et s'excuser auprès des personnes affectées par nos erreurs et apprendre de ces expériences
+* Se concentrer sur ce qui est le meilleur non pas uniquement pour nous en tant qu'individu, mais pour la communauté
+
+Exemples de comportements non acceptables :
+
+* L'utilisation de langage ou d'imageries sexualisés et les avances sexuelles non

```

Comment:
```suggestion
* L'utilisation d’un quelconque language sexuel et les avances sexuelles non

```

Reply:
“Imagerie” is old French and not commonly used anymore.

Reply:
```suggestion
* L'utilisation de langage ou d'images sexualisés et d'attentions ou d'avances sexuelles de toute nature
```

Reply:
I think we should keep something about the "imagery" that was mentionned in the english version too.
Nevertheless, the word "imagerie" is bit old indeed.